File: PokeIn/PokeIn_Free_v2.032/PokeIn_Free_v2.032/VB.NET MVC3 Razor/VB.NET MVC3/Controllers/HomeController.vb
Public Class HomeController
    Inherits System.Web.Mvc.Controller

    Function Index() As ActionResult
        ViewData("Message") = "Welcome to ASP.NET MVC! (PokeIn Sample)"

        Return View()
    End Function

    Function About() As ActionResult
        Return View()
    End Function

    'PokeIn: Handler page is defined here
    Function Handler() As ActionResult
        Return View()
    End Function
End Class

File: PokeIn/PokeIn_Free_v2.032/PokeIn_Free_v2.032/MVC3 Razor CS/MVC3 Razor CS/Controllers/HomeController.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using System.Web.Mvc;

namespace MVC3_Razor_CS.Controllers
{
    public class HomeController : Controller
    {
        public ActionResult Index()
        {
            ViewBag.Message = "Welcome to ASP.NET MVC! (PokeIn Sample)";

            return View();
        }

        public ActionResult About()
        {
            return View();
        }

        //PokeIn: Handler page is defined here
        public ActionResult Handler()
        {
            return View();
        }
    }
}
